Formula/survex.rb

```diff
@@ -0,0 +1,72 @@
+class Survex < Formula
+  desc "Cave Surveying Tool"
+  homepage "https://www.survex.com"
+  url "https://survex.com/software/1.2.32/survex-1.2.32.tar.gz"
+  sha256 "ac15252618f6a59617ec1142609f04bba69b1f18b3490729e405ff89a4b22852"
+
+  depends_on "wxmac"
+  depends_on "proj"
+  depends_on "ffmpeg"
+
+  depends_on "gettext" => :build
+
+  def install
+    # Allow for non-static linking against PROJ EPSG/ESRI definitions
+    ["src/cavern.c", "src/aven.cc"].each { |f| inreplace f, /pj_set_finder\(msg_proj_finder\)/, "" }
```

Comment:
This needs to be fixed in the upstream project. We try to minimise the patching done in Homebrew.